hack/generate-configmap.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# This script generates configmap/s set by arguments
+#
+# To create a CI configmap use the "--ci" option
+# To create an RHEL configmap use the "--rhel" option
+# To create both configmaps use the "--all" option or pass no arguments
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Call the function teardown() on the exit of the script
+trap teardown EXIT
+
+teardown() {
+	rm -rf "${TEMP_DIR}"
+}
+
+TEMP_DIR=$(mktemp -d -t keys-XXXXXXXX)
+CONFIGMAP_FILENAME="0000_90_cluster-update-keys_configmap.yaml"
+
+# Generates a public key specified by parameters in the TEMP_DIR directory
+#
+# $1 - Name of the key which will be generated
+# $2 - Verifier public keys separated by internal field separator
+#
+generate_public_key() {
+	truncate -s 0 "${TEMP_DIR}/key.gpg" # Empty the ${TEMP_DIR}key.gpg file
```

Comment:
nit: `truncate` doesn't seem to be [in POSIX][1].  Can we use:

```sh
> "${TEMP_DIR}/key.gpg"
```

or:

```sh
rm -f "${TEMP_DIR}/key.gpg"
```

or some such instead?  On the other hand, it's not like tons of folks are likely to be running this script, so portable POSIX compliance isn't that big a win.

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/idx/utilities.html

Reply:
Good catch! I tried to make the bash script POSIX compliant. However I must have ended up being too dependent on a POSIX shell check tool. Thanks!

Fixed in https://github.com/openshift/cluster-update-keys/commit/d8197c0d0a485af7bb0b9f423e461f9606989991.